vtest/scores/bravura-10.mscx

```diff
@@ -1,30 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<museScore version="3.01">
-  <programVersion>3.0.0</programVersion>
-  <programRevision>c07ed54</programRevision>
+<museScore version="3.02">
```

Comment:
Why have you modified all these vtests?

Reply:
As mentioned in the commit (modulo typos):
"chaning their page size to fit their content, but otherwise keeping pre 3.6 style"

Reply:
And to fix the vtests, see https://github.com/musescore/MuseScore/pull/23473#issuecomment-2212501079

Reply:
I still don't understand why you changed them (besides, your commit mentiones 3 tests by there are 13 changed files). Are the vtests now passing because your PR is not affecting them, or because you changed them?

Reply:
That commit changes 3 vtest scores. And I'm not really sure why they pass now, to be honest, whether it is because of the size change or because of the version change, or both or something different.

Other vtest scores had been changed as per @shoogle's comment for Step 2, https://github.com/musescore/MuseScore/pull/23473#pullrequestreview-2161851516

Reply:
Hmm I guess it is the removed 
```xml
      <staffTextPosAbove>-1</staffTextPosAbove>
```
that makes the tests pass

Reply:
I'm sorry, but that doesn't make any sense. 
a) We do NOT "fix" the vtest files. Those files are there for the exact purpose of telling us how the program changes in time: if we change the files and update them to what the program does now we are making them completely pointless!! 
b) If a vtest file contains a typo it's very important that _the typo stays there_, because it represents what the program did at that time, which means that there are likely thousands of user scores with that same typo out there, and that test is telling us exactly how the _current_ Musescore reacts to that old typo, which is an important information. 
c) If we do decide to update certain vtests files, we do it in a separate PR, othewise we don't know anymore if the PR is passing/failing the tests, because we are changing the reference of it.

Reply:
Those vtests that stemmed from Mu 4.4.0, needed to get adjusted to the changed XML tags of 4.4.0 (changed in this PR).

Those 3 vtest scores that stemmed from Mu 3.0.0 apparently contained an bogus/outdated(?) style tag, `staffTextPosABove`, for sure one that got ignored before this PR, but does get interpreted now, and then broke the vtests.
Instead of changing those 3 vtests, we could ignore `staffTextPosAbove` for pre 4.4.0 (or whenever that style setting really got removed) scores, and/or read them only for 4.4.0 and later scores.
These 3 vtests should get changed anyhow, for their content to fit their page size, but indeed that could be done via a separate PR.

Reply:
Ah ok, then yes _only_ for the 4.4 files it is ok to fix the typos.

About the other 3 tests: the one you showed in the comment above seems an improvement to me. And also, staffTextPosAbove doesn't seem like a wrong style tag, it probably made sense at the time that it was created. So I think it's actually an improvement if we now read it and consequently do a better placement of those text items (in other words, having those 3 tests fail may be an improvement, that's I'd like to see them fail rather than change the files)

Reply:
I gust did that, please recheck once the tests passed

Reply:
Or change these 3 to 3.6 scores and use the then valid `staffPosAbove` and ignore the resulting vtest fails?

Reply:
Or do that in a separate PR?

Reply:
Or niether ignore that old tag nor fix the vtests, but just accept the fails
(and adjust just the size in a separate PR)?

Reply:
Those 3 failing vtests look like an improvement for me. Are they not?

Reply:
They may be seen as such, yes. Esp. now as I (sort of) understand why...

Reply:
I just wonder a) when exactly that had been introduced and b) why back then the vtests fail had been ignored

Reply:
If so, just add the vtests label ;-)

Reply:
Seems the cause is `<staffTextMinDistance>0.2</staffTextMinDistance>` in that score

Reply:
Boh are now read in explicitly, but at the same place we could instead ignore them.
Ignoring them and have the vtests pass or read and accept the vtests fails

Reply:
Only quirk is that these scores get report as `m_version == 410` rather than `301`, not sure at all why.

Reply:
Oh well, finally there... here we go, vtests 'failing' as per https://github.com/musescore/MuseScore/pull/23473#issuecomment-2212501079